plugins/tileacq.py

```diff
@@ -691,7 +694,9 @@ def acquire(self, dlg):
                     return
 
                 # Sort tiles (largest sem on first position)
-                da_list.append(self.sort_das(das, ss))
+                if self.sort_das(das, ss):
```

Comment:
This whole code doesn't change anything. Appending an empty list is a noop anyway. => revert to original simpler code.

Reply:
if I append a list with an empty list, then the check "if da_list" doesn't work anymore (it will return True if the list contains an empty list, but it should return False)

Reply:
Sorry, you're right, of course! I mixed up with .extend(). Still, let's avoid the `if`. Would it just work to write `if da_list[0]`? Or, if da_list might itselft be empty then you could put `if da_list and da_list[0]`.

Reply:
ok, I'll change it